### PR TITLE
Make version copiable

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-web/issues/17603
Fixes https://github.com/vector-im/element-web/issues/18329

<hr>

Screenshot_20210802_113722

### CODE REVIEW

**Comment by aaronraimist** [comment]
Might want to capitalize Olm while you are editing this

File: src/components/views/settings/tabs/user/HelpUserSettingsTab.tsx [hunk 1]
```diff
@@ -68,6 +68,18 @@ export default class HelpUserSettingsTab extends React.Component<IProps, IState>
...
 
+    private getVersionInfo(): { appVersion: string, olmVersion: string } {
+        const brand = SdkConfig.get().brand;
+        const appVersion = this.state.appVersion || 'unknown';
+        let olmVersion = MatrixClientPeg.get().olmVersion;
+        olmVersion = olmVersion ? `${olmVersion[0]}.${olmVersion[1]}.${olmVersion[2]}` : '<not-enabled>';
+
+        return {
+            appVersion: `${_t("%(brand)s version:", { brand })} ${appVersion}`,
+            olmVersion: `${_t("olm version:")} ${olmVersion}`,
```
Comment (SimonBrandner) [inline src/components/views/settings/tabs/user/HelpUserSettingsTab.tsx:79]:
```suggestion
            olmVersion: `${_t("Olm version:")} ${olmVersion}`,
```

**Review by t3chguy — APPROVED** [review]
Code wise LGTM

**Comment by SimonBrandner** [comment]
> This feels a bit far away, and to me looks like it's copying _all_ of the debug info, wouldn't it be possible to have the button right next to the version indicator? And not on the far right of the element?

I can probably make the whole area a bit smaller

**Comment by ShadowJonathan** [comment]
Just saw the new screenshot, this looks much better, thank you!

The only thing I'd be worried about is overflow of the version into the next line when it gets too long, but that's minor.

**Review by niquewoodhouse — APPROVED** [review]
My only thought was having separate copy buttons for each item, but I'm not that familiar with the area and I think this is probably easier to understand for most people.  It fits in really well with the pre-existing UI too.

Thanks for contributing!

**Pull request merged**